python/samples/concepts/memory/vector_record_store_class.py

```diff
@@ -0,0 +1,110 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+
+from dataclasses import dataclass, field
+from typing import Annotated
+from uuid import uuid4
+
+from numpy import array
+
+from semantic_kernel import Kernel
+from semantic_kernel.connectors.ai.open_ai.prompt_execution_settings.open_ai_prompt_execution_settings import (
+    OpenAIEmbeddingPromptExecutionSettings,
+)
+from semantic_kernel.connectors.ai.open_ai.services.open_ai_text_embedding import OpenAITextEmbedding
+from semantic_kernel.connectors.memory.azure_ai_search.azure_ai_search_collection import (
+    AzureAISearchCollection,
+)
+from semantic_kernel.data.vector_store_model_decorator import vectorstoremodel
+from semantic_kernel.data.vector_store_record_fields import (
+    VectorStoreRecordDataField,
+    VectorStoreRecordKeyField,
+    VectorStoreRecordVectorField,
+)
+
+
+@vectorstoremodel
+@dataclass
+class MyDataModel:
+    vector: Annotated[
+        list[float] | None,
+        VectorStoreRecordVectorField(
+            embedding_settings={"embedding": OpenAIEmbeddingPromptExecutionSettings(dimensions=1536)},
+            index_kind="hnsw",
+            dimensions=1536,
+            distance_function="cosine",
+            property_type="list[float]",
+            # cast_function=array,
+        ),
+    ] = None
+    other: str | None = None
+    id: Annotated[str, VectorStoreRecordKeyField()] = field(default_factory=lambda: str(uuid4()))
+    content: Annotated[
+        str, VectorStoreRecordDataField(has_embedding=True, embedding_property_name="vector", property_type="str")
+    ] = "content1"
+
+
+kernel = Kernel()
+
+
+stores = {
+    "ai_search": AzureAISearchCollection[MyDataModel](
+        data_model_type=MyDataModel,
+        kernel=kernel,
+    ),
+    # "redis": RedisVectorRecordStore[MyDataModel](
+    #     data_model_type=MyDataModel,
+    #     kernel=kernel,
+    #     collection_name="test",
+    #     prefix_collection_name_to_key_names=True,
+    # ),
+    # "qdrant": QdrantVectorRecordStore[MyDataModel](
+    #     data_model_type=MyDataModel, kernel=kernel, collection_name="test", prefer_grpc=True
+    # ),
+}
+
+store = "ai_search"
```

Comment:
nit: a comment on how to use this variable with the dict above to try a different vector store.

Reply:
nit: Just a style thing but I find it easier to read when file level configuration variables like this are at the top of the file (under the imports) and preferably commented with a `#configuration` or something like that.